samples/core-projects/libraries/README.md

```diff
@@ -0,0 +1,83 @@
+# .NET Core Library Samples
+
+**Note:** These are compatibile with tooling that is still in active development.  Details are subject to change over time.
+
+These samples are buildable projects whose source are used for code snippets in [the guide for writing cross-platform libraries](../../../docs/libraries/libraries-with-cli.md).  They can be built and run using the .NET Core RC1 toolchain.
+
+They are small and intended to demonstrate how to target and build NuGet packages for different targets.
+
+To build/use any of these (using `new-library` as an example):
+
+1. Open your favorite Command Line Interface.
+
+2. Navigate to the top-level directory:
+
+	`$ cd new-library`
+
+3. Restore packages by typing the following:
+
+	`$ dnu restore -s "https://myget.org/F/xunit"`
+	
+	`$ dnu restore`
+	
+	Note: The first `dnu restore` is needed right not to allow for unit tests to run.  This will not be required in the future.
+
+4. To build and package the library as a NuGet package, type the following:
+
+	`$ cd src/Library`
+	
+	`$ dnu build`
+	
+	`$ dnu pack`
+	
+	Check out the `/bin/Debug` directory to see the generated artifacts and `.nupkg`.
+
+5. To run unit tests (only applicable to `new-library`):
+
+	`$ cd ../../test/LibraryTests`
+	
+	`$ dnx test`
+
+And that's it!
+
+## new-library
+
+The project under `/new-library` is a project targeting **only** .NET Core.  Backwards compatibility is not a goal.
+
+It also showcases two other things: how to use multiple projects and how to test.
```

Comment:
Slightly odd. This should be more uniform. Have one way of talking about everything this sample demonstrates. Not, "here are the two other things it showcases."

I prefer the term "demonstrates" to "showcases". It feels like a more clear term.

I doubt "showcased" is really the term you want to use for the "two libraries". Are they simply "used"?